tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -435,6 +435,86 @@ test.describe( 'Nested Tabs tests', () => {
 
 		await cleanup( wpAdmin );
 	} );
+
+	test( 'Check that active tab receives custom hover color when hovering above it', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+
+		await setup( wpAdmin );
+
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Add widgets.
+		await editor.addWidget( 'nested-tabs', container );
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tab-title.e-normal.e-active' );
+
+		// Act.
+		// Set tab hover color.
+		await setTabItemColor( page, editor, 'tabs', 'tabs_title_hover', 'tabs_title_background_color_hover_color', '#ff0000' );
+
+		await editor.publishAndViewPage();
+
+		// Hover background style.
+		const backgroundHoverStyle = 'rgb(255, 0, 0)',
+			defaultActiveTabColor = 'rgb(97, 206, 112)',
+			activeTab = await page.locator( '.e-normal.e-active' );
+
+		// Assert.
+		// Check that active tab default color is seen.
+		await expect( activeTab ).toHaveCSS( 'background-color', defaultActiveTabColor );
+
+		await activeTab.hover();
+
+		// Check that active tab receives the hover defined background color.
+		await expect( activeTab ).toHaveCSS( 'background-color', backgroundHoverStyle );
+
+		await cleanup( wpAdmin );
+	} );
+
+	test( 'Check that while hovering an active tab hover styling is shown', async ( { page }, testInfo ) => {
```

Comment:
Suggestion: 'Check if the hover effect works on active tabs`

Reply:
Great suggestion.
Done.